src/avatar.jsx

```diff
@@ -86,6 +83,10 @@ const Avatar = React.createClass({
     this.setState({muiTheme: newMuiTheme});
   },
 
+  getTheme() {
```

Comment:
I agree with the removal of `getTheme` it seems really redundant.

>  we do have to traverse down a lot of nested properties to get what we need

It's not us it's javascript :laughing: :laughing: although nested destructuring takes some of the pain away.